garage/tf/core/networks.py

```diff
@@ -0,0 +1,71 @@
+import tensorflow as tf
+from garage.core import Serializable
+
+
+class MLPs(Serializable):
```

Comment:
I think it would be better if we can abstract out a model with appropriate API's for different models like GaussianMLP, LSTM and etc.